phpunit.xml

```diff
@@ -1,24 +1,13 @@
-<phpunit
-  bootstrap="vendor/autoload.php"
-  colors="true"
-  convertErrorsToExceptions="true"
-  convertWarningsToExceptions="true"
-  stopOnError="false"
-  stopOnFailure="false"
-  stopOnIncomplete="false"
-  stopOnSkipped="false"
-  verbose="false">
-
+<?xml version="1.0"?>
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" bootstrap="vendor/autoload.php" colors="true" convertErrorsToExceptions="true" convertWarningsToExceptions="true" stopOnError="false" stopOnFailure="false" stopOnIncomplete="false" stopOnSkipped="false" verbose="false" xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.3/phpunit.xsd">
```

Comment:
Can you restore the previous formatting? This makes it easier to keep track of future changes.